lavalink/rest_api.py

```diff
@@ -211,9 +211,13 @@ def __init__(self, data):
                 ):
                     continue
                 elif k == "exception":
-                    v["message"] = v["message"] + "\n{query}\n{response}".format(
-                        query="Query: " + data["encodedquery"] if data.get("encodedquery") else "",
-                        response=str(self._raw),
+                    v["message"] = (
+                        f"Timestamp: {self._raw.get('timestamp', 'Unknown')}\n"
+                        f"Status Code: {self._raw.get('status', 'Unknown')}\n"
+                        f"Error: {self._raw.get('error', 'Unknown')}\n"
+                        f"query: {self._raw.get('query', 'Unknown')}\n"
+                        f"loadType: {self._raw.get('loadType', 'Unknown')}\n"
+                        f"message: {self._raw.get('message', v['message'])}"
```

Comment:
Casing and "loadType" key will always be there cause of the `self._fallback` so `.get()` was unnecessary
```suggestion
                        f"Query: {self._raw.get('query', 'Unknown')}\n"
                        f"Load Type: {self._raw['loadType']}\n"
                        f"Message: {self._raw.get('message', v['message'])}"
```